ports/copypp/portfile.cmake

```
@@ -0,0 +1,25 @@
+set(VCPKG_BUILD_TYPE release) # header-only
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO i-curve/copypp
+    REF "v${VERSION}"
+    SHA512 a13a3ee105b3802f585c086846dbce7d5e9c23d11bf55fcc0c63e1a4b21fc6798f1a324cfb3da66b715dfc22a4f1f7ba51a2f4e10db331f7d4fd6b20a9be6d41
+    HEAD_REF main
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS "-DCOPYPP_TEST=OFF"
+)
+
+vcpkg_cmake_install()
+
+vcpkg_fixup_cmake_targets(CONFIG_PATH lib/cmake/copypp)
```

```suggestion
vcpkg_cmake_config_fixup(CONFIG_PATH lib/cmake/copypp)
```

Sorry that I didn't notice this earlier. The old function has been deprecated a long time ago, there should be a warning in build output, and there should be a warning here on GH... Not sure how this slipped through.